doc/Addons.md

```diff
@@ -264,7 +264,7 @@ $data = [
 ##### With multiple submit buttons
 ```php
 $data = [
-	'addon'  => 'catavar',
+	'addon'  => 'catavatar',
```

Comment:
I'm pretty sure this is https://git.friendi.ca/friendica/friendica-addons/src/commit/49254a8307f80cfe3ebd165ca556c3acb893b7c6/catavatar

Reply:
Good catch!